python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -527,6 +539,24 @@ def get_partition_mapping(self, in_asset_key: AssetKey) -> PartitionMapping:
                 else AllPartitionMapping(),
             )
 
+    def with_freshness_policies(
```

Comment:
hm fair point -- in my opinion the main utility of this would come from calling this repeatedly on a list of assets (similarly to the with_resources() function vs the AssetsDefinition.with_resources method). So you could load a bunch of assets from_package_module and apply the same freshness policy to all of them. This doesn't seem particularly important though so I'll take this out for now.